test/test_common.c

```diff
@@ -219,44 +258,58 @@ sport_new (const char *optarg, struct prog *prog)
     sport->packs_in = NULL;
     sport->fd = -1;
     char *const addr = strdup(optarg);
+    //char ip[INET6_ADDRSTRLEN];
 #if __linux__
     char *if_name;
     if_name = strrchr(addr, ',');
     if (if_name)
     {
         strncpy(sport->if_name, if_name + 1, sizeof(sport->if_name) - 1);
-        sport->if_name[ sizeof(sport->if_name) - 1 ] = '\0';
+        sport->if_name[sizeof(sport->if_name) - 1] = '\0';
```

Comment:
Another whitespace-only change.

Reply:
An easy way to get a diff is to add `.diff` to the PR URI:

https://github.com/litespeedtech/lsquic-client/pull/30.diff